### PR TITLE
lab-01

### PR DESCRIPTION
**Pull request by AariyankaPriyanka**

### CODE REVIEW

**Review by kasthurimahadeva — COMMENTED** [review]
Q1 - 05

**Review by kasthurimahadeva — COMMENTED** [review]
Q2-10

**Review by kasthurimahadeva — COMMENTED** [review]
Q3-10

**Review by kasthurimahadeva — COMMENTED** [review]
Q4-10

**Review by kasthurimahadeva — COMMENTED** [review]
Q5-15

**Review by kasthurimahadeva — COMMENTED** [review]
Q8-10

**Review by kasthurimahadeva — COMMENTED** [review]
Q9-15

**Review by kasthurimahadeva — COMMENTED** [review]
Q10-15

File: lab-01/QUESTION-06.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <ol>
+                    <li>Borella</li>
+                    <li>Kotte</li>
+                
+                <ol type="a" start="2">
+                    <li>Pita Kotte</li>
+                    <li>Ethul Kotte</li>
+                </ol>
+                <li>Nugegoda</li>
+                <li>Maharagama</li>
```
Comment (kasthurimahadeva) [inline lab-01/QUESTION-06.html]: check this line

**Comment by AariyankaPriyanka** [comment]
<html>
    <head>
        <title>QUESTION-06</title>
    </head>
    <body>
        <ol type="A" start="2">
            <li>Western province</li>
            <ol type="I" start="1">
                <li>Colombo</li>
                <ol>
                    <li>Borella</li>
                    <li>Kotte</li>
                
                <ol type="a" start="2">
                    <li>Pita Kotte</li>
                    <li>Ethul Kotte</li>
                </ol>
                <li>Nugegoda</li>
                <li>Maharagama</li>
                </ol>
                <li>Gampaha</li>
            </ol>
            <li>North western Province</li>
            <ol type="i" start="3">
                <li>Kurunegala</li>
                <li>Puttlam</li>
            </ol>
        </ol>
    </body>
</html>

**Review by kasthurimahadeva — COMMENTED** [review]
Q6-10

**Comment by kasthurimahadeva** [comment]
Lab1 => 100/100

**Comment by kasthurimahadeva** [comment]
100/100

**Pull request closed without merging**